specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-12-01-preview/definitions.json

```diff
@@ -4010,6 +4010,180 @@
       },
       "description": "A collection of resources."
     },
+    "PortalConfigCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PortalConfigContract"
+          },
+          "description": "The developer portal configurations."
+        },
+        "count": {
```

Comment:
nit: Generally most service's collection get does not return a total count as that can change at anytime thus render the count useless. This could potentially also adds to the service processing with a count() call instead of top() if you are backed by SQL.
Typical pattern is just supply a next link allow user to do repeatable GETs until it returns no more rows.

Reply:
Makes sense.